code/modules/vending/clothesmate.dm

```diff
@@ -33,9 +33,15 @@
 		            /obj/item/clothing/neck/scarf/black = 3,
 		            /obj/item/clothing/neck/scarf/zebra = 3,
 		            /obj/item/clothing/neck/scarf/christmas = 3,
-		            /obj/item/clothing/neck/stripedredscarf = 3,
-		            /obj/item/clothing/neck/stripedbluescarf = 3,
-		            /obj/item/clothing/neck/stripedgreenscarf = 3,
+		            /obj/item/clothing/neck/yogs/stripedscarf/black = 3, //yogs start
```

Comment:
Move these scarfs at the end of the clothes mate file where all the other yogs stuff is for max modularization

Reply:
donso